### DISCUSSION TITLE
Russian TV to Found Names

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#218] [Show and tell]
http://188.113.190.14/1/video.m3u8
http://188.113.190.14/2/video.m3u8
http://188.113.190.14/3/video.m3u8
http://188.113.190.14/4/video.m3u8
http://188.113.190.14/5/video.m3u8
http://188.113.190.14/6/video.m3u8
http://188.113.190.14/7/video.m3u8
http://188.113.190.14/8/video.m3u8
http://188.113.190.14/9/video.m3u8
http://188.113.190.14/10/video.m3u8
http://188.113.190.14/11/video.m3u8
http://188.113.190.14/12/video.m3u8
http://188.113.190.14/13/video.m3u8
http://188.113.190.14/14/video.m3u8
http://188.113.190.14/15/video.m3u8
http://188.113.190.14/16/video.m3u8
http://188.113.190.14/17/video.m3u8
http://188.113.190.14/18/video.m3u8
http://188.113.190.14/19/video.m3u8
http://188.113.190.14/20/video.m3u8
http://188.113.190.14/21/video.m3u8
http://188.113.190.14/22/video.m3u8
http://188.113.190.14/23/video.m3u8
http://188.113.190.14/24/video.m3u8
http://188.113.190.14/25/video.m3u8 off
http://188.113.190.14/26/video.m3u8
http://188.113.190.14/27/video.m3u8 off
http://188.113.190.14/28/video.m3u8
http://188.113.190.14/29/video.m3u8
http://188.113.190.14/30/video.m3u8
http://188.113.190.14/31/video.m3u8
http://188.113.190.14/32/video.m3u8
http://188.113.190.14/33/video.m3u8
http://188.113.190.14/34/video.m3u8
http://188.113.190.14/35/video.m3u8
http://188.113.190.14/36/video.m3u8
http://188.113.190.14/37/video.m3u8
http://188.113.190.14/38/video.m3u8
http://188.113.190.14/39/video.m3u8
http://188.113.190.14/40/video.m3u8
http://188.113.190.14/41/video.m3u8
http://188.113.190.14/42/video.m3u8
http://188.113.190.14/43/video.m3u8
http://188.113.190.14/44/video.m3u8
http://188.113.190.14/45/video.m3u8
http://188.113.190.14/46/video.m3u8
http://188.113.190.14/47/video.m3u8
http://188.113.190.14/48/video.m3u8
http://188.113.190.14/49/video.m3u8
http://188.113.190.14/50/video.m3u8
http://188.113.190.14/51/video.m3u8
http://188.113.190.14/52/video.m3u8
http://188.113.190.14/53/video.m3u8
http://188.113.190.14/54/video.m3u8
http://188.113.190.14/55/video.m3u8
http://188.113.190.14/56/video.m3u8
http://188.113.190.14/57/video.m3u8
http://188.113.190.14/58/video.m3u8
http://188.113.190.14/59/video.m3u8
http://188.113.190.14/60/video.m3u8
http://188.113.190.14/61/video.m3u8
http://188.113.190.14/62/video.m3u8
http://188.113.190.14/63/video.m3u8
http://188.113.190.14/64/video.m3u8
http://188.113.190.14/65/video.m3u8
http://188.113.190.14/66/video.m3u8
http://188.113.190.14/67/video.m3u8
http://188.113.190.14/68/video.m3u8
http://188.113.190.14/69/video.m3u8
http://188.113.190.14/70/video.m3u8
http://188.113.190.14/71/video.m3u8
http://188.113.190.14/72/video.m3u8
http://188.113.190.14/73/video.m3u8
http://188.113.190.14/74/video.m3u8
http://188.113.190.14/75/video.m3u8
http://188.113.190.14/76/video.m3u8
http://188.113.190.14/77/video.m3u8
http://188.113.190.14/78/video.m3u8
http://188.113.190.14/79/video.m3u8
http://188.113.190.14/80/video.m3u8
http://188.113.190.14/81/video.m3u8
http://188.113.190.14/82/video.m3u8
http://188.113.190.14/83/video.m3u8
http://188.113.190.14/84/video.m3u8
http://188.113.190.14/85/video.m3u8
http://188.113.190.14/86/video.m3u8
http://188.113.190.14/87/video.m3u8
http://188.113.190.14/88/video.m3u8
http://188.113.190.14/89/video.m3u8
http://188.113.190.14/90/video.m3u8
http://188.113.190.14/91/video.m3u8
http://188.113.190.14/92/video.m3u8
http://188.113.190.14/93/video.m3u8
http://188.113.190.14/94/video.m3u8
http://188.113.190.14/95/video.m3u8
http://188.113.190.14/96/video.m3u8
http://188.113.190.14/97/video.m3u8
http://188.113.190.14/98/video.m3u8
http://188.113.190.14/99/video.m3u8
http://188.113.190.14/100/video.m3u8
http://188.113.190.14/101/video.m3u8
http://188.113.190.14/102/video.m3u8
http://188.113.190.14/103/video.m3u8
http://188.113.190.14/104/video.m3u8
http://188.113.190.14/105/video.m3u8
http://188.113.190.14/106/video.m3u8
http://188.113.190.14/107/video.m3u8
http://188.113.190.14/108/video.m3u8
http://188.113.190.14/109/video.m3u8
http://188.113.190.14/110/video.m3u8
http://188.113.190.14/111/video.m3u8
http://188.113.190.14/112/video.m3u8
http://188.113.190.14/113/video.m3u8
http://188.113.190.14/114/video.m3u8
http://188.113.190.14/115/video.m3u8 OFF
http://188.113.190.14/116/video.m3u8
http://188.113.190.14/117/video.m3u8
http://188.113.190.14/118/video.m3u8
http://188.113.190.14/119/video.m3u8
http://188.113.190.14/120/video.m3u8
http://188.113.190.14/121/video.m3u8
http://188.113.190.14/122/video.m3u8
http://188.113.190.14/123/video.m3u8
http://188.113.190.14/124/video.m3u8
http://188.113.190.14/125/video.m3u8
http://188.113.190.14/126/video.m3u8
http://188.113.190.14/127/video.m3u8
http://188.113.190.14/128/video.m3u8
http://188.113.190.14/129/video.m3u8
http://188.113.190.14/130/video.m3u8
http://188.113.190.14/131/video.m3u8
http://188.113.190.14/132/video.m3u8
http://188.113.190.14/133/video.m3u8
http://188.113.190.14/134/video.m3u8
http://188.113.190.14/135/video.m3u8
http://188.113.190.14/136/video.m3u8
http://188.113.190.14/137/video.m3u8
http://188.113.190.14/138/video.m3u8
http://188.113.190.14/139/video.m3u8
http://188.113.190.14/140/video.m3u8
http://188.113.190.14/141/video.m3u8 OFF
 http://188.113.190.14/142/video.m3u8
http://188.113.190.14/143/video.m3u8
http://188.113.190.14/144/video.m3u8
http://188.113.190.14/145/video.m3u8
http://188.113.190.14/146/video.m3u8
http://188.113.190.14/147/video.m3u8
http://188.113.190.14/148/video.m3u8
http://188.113.190.14/149/video.m3u8
http://188.113.190.14/150/video.m3u8
http://188.113.190.14/151/video.m3u8
http://188.113.190.14/152/video.m3u8
http://188.113.190.14/153/video.m3u8
http://188.113.190.14/154/video.m3u8
http://188.113.190.14/155/video.m3u8
http://188.113.190.14/156/video.m3u8
http://188.113.190.14/157/video.m3u8
http://188.113.190.14/158/video.m3u8
http://188.113.190.14/159/video.m3u8
http://188.113.190.14/160/video.m3u8
http://188.113.190.14/161/video.m3u8
http://188.113.190.14/162/video.m3u8
http://188.113.190.14/163/video.m3u8
http://188.113.190.14/164/video.m3u8
http://188.113.190.14/165/video.m3u8
http://188.113.190.14/166/video.m3u8
http://188.113.190.14/167/video.m3u8
http://188.113.190.14/168/video.m3u8
http://188.113.190.14/169/video.m3u8
http://188.113.190.14/170/video.m3u8
http://188.113.190.14/171/video.m3u8
http://188.113.190.14/172/video.m3u8
http://188.113.190.14/173/video.m3u8
http://188.113.190.14/174/video.m3u8
http://188.113.190.14/175/video.m3u8
http://188.113.190.14/176/video.m3u8
http://188.113.190.14/177/video.m3u8
http://188.113.190.14/178/video.m3u8
http://188.113.190.14/179/video.m3u8
http://188.113.190.14/180/video.m3u8
http://188.113.190.14/181/video.m3u8
http://188.113.190.14/182/video.m3u8
http://188.113.190.14/183/video.m3u8
http://188.113.190.14/184/video.m3u8
http://188.113.190.14/185/video.m3u8
http://188.113.190.14/186/video.m3u8
http://188.113.190.14/187/video.m3u8
http://188.113.190.14/188/video.m3u8
http://188.113.190.14/189/video.m3u8
http://188.113.190.14/190/video.m3u8
http://188.113.190.14/191/video.m3u8 OFF
http://188.113.190.14/192/video.m3u8
http://188.113.190.14/193/video.m3u8
http://188.113.190.14/194/video.m3u8
http://188.113.190.14/195/video.m3u8
196 to 200 = OFF
http://188.113.190.14/201/video.m3u8
http://188.113.190.14/202/video.m3u8
http://188.113.190.14/203/video.m3u8
http://188.113.190.14/204/video.m3u8
http://188.113.190.14/205/video.m3u8
http://188.113.190.14/206/video.m3u8
http://188.113.190.14/207/video.m3u8
http://188.113.190.14/208/video.m3u8
http://188.113.190.14/209/video.m3u8
http://188.113.190.14/210/video.m3u8
http://188.113.190.14/211/video.m3u8
http://188.113.190.14/212/video.m3u8
http://188.113.190.14/213/video.m3u8
http://188.113.190.14/214/video.m3u8
http://188.113.190.14/215/video.m3u8
http://188.113.190.14/216/video.m3u8
http://188.113.190.14/217/video.m3u8
http://188.113.190.14/218/video.m3u8
http://188.113.190.14/219/video.m3u8
http://188.113.190.14/220/video.m3u8
http://188.113.190.14/221/video.m3u8
http://188.113.190.14/222/video.m3u8
http://188.113.190.14/223/video.m3u8
http://188.113.190.14/224/video.m3u8
http://188.113.190.14/225/video.m3u8
http://188.113.190.14/226/video.m3u8
http://188.113.190.14/227/video.m3u8
http://188.113.190.14/228/video.m3u8
http://188.113.190.14/229/video.m3u8
http://188.113.190.14/230/video.m3u8
http://188.113.190.14/231/video.m3u8
http://188.113.190.14/232/video.m3u8 OFF
http://188.113.190.14/233/video.m3u8
http://188.113.190.14/234/video.m3u8 OFF
http://188.113.190.14/235/video.m3u8
http://188.113.190.14/236/video.m3u8 OFF
to 300 = OFF

**Comment by BellezaEmporium** [comment]
Gonna check links using my script, and modify over your comment.

EDIT : Found channels around 301 to 337.
[188.113.190.14.zip](https://github.com/iptv-org/iptv/files/7448459/188.113.190.14.zip)

There's some Digital Television Service system covering name (and maybe EPG ? idk). Got broken characters because I do not have the Cyrillic alphabet in my computer.
